shell/common/shell.cc

```diff
@@ -40,6 +40,19 @@ constexpr char kSystemChannel[] = "flutter/system";
 constexpr char kTypeKey[] = "type";
 constexpr char kFontChange[] = "fontsChange";
 
+// Add the original asset directory to the resolvers so that unmodified assets
+// bundled with the application specific format (APK, IPA) can be used without
+// syncing to the Dart devFS.
+std::unique_ptr<DirectoryAssetBundle> restoreOriginalAssetResolver(
```

Comment:
Make this a private method in `Shell` that can access `settings_`

Also capitalize the first letter in the method name.